stock_warehouse_partner_security/i18n/es.po

```diff
@@ -21,11 +21,15 @@ msgid "Inventoried Quantity"
 msgstr "Cantidad inventariada"
 
 #. module: stock_warehouse_partner_security
-#: code:addons/slv-stock-chr/stock_warehouse_partner_security/models/stock_quant.py:0
+#: model:ir.actions.server,name:stock_warehouse_partner_security.action_view_quants
+msgid "Inventory (in stock_warehouse_partner_security)"
+msgstr ""
```

Comment:
Missing translate

Reply:
This name belongs to a server action that is hidden in Inventory model, because is directly called from the menu (https://github.com/odoo/odoo/blob/13.0/addons/stock/views/stock_quant_views.xml#L156), so that description is never shown. Anyway, I'll translate it since is available through technical features server actions menu.